plugins/modules/packaging/os/zypper_repository.py

```diff
@@ -143,7 +143,10 @@ def _parse_repos(module):
     """parses the output of zypper --xmlout repos and return a parse repo dictionary"""
     cmd = _get_cmd('--xmlout', 'repos')
 
-    from xml.dom.minidom import parseString as parseXML
+    try:
+        from xml.dom.minidom import parseString as parseXML
+    except ImportError:
+        module.fail_json(msg="python-xml is a requirement for zypper_repository module")
```

Comment:
Please use https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L646 to format the message.

Reply:
Fixed in https://github.com/ansible-collections/community.general/pull/939/commits/305dd924bc75b3fcc6d31d502f7b4ccc401ab67b.